examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
<p>Intro and refs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yeah, still need to add these